### PR TITLE
Made aggregate() and datashade() compute plot size automatically

### PR DESCRIPTION
**Pull request by jbednar**
Seems like nearly everyone will want this, and it's easy enough to override if you don't.

### CODE REVIEW

**Comment by jlstevens** [comment]
Looks good! 

Tests are still running as Travis is being pretty slow today and I think there is an open issue to start testing the datashader operations. Merge now or wait for Travis?

**Comment by jbednar** [comment]
@philippjfr, if this looks ok to you, then I think it's ready to merge.

File: holoviews/operation/datashader.py [hunk 1]
```diff
@@ -81,9 +81,14 @@ class aggregate(ElementOperation):
 
     The bins of the aggregate are defined by the width and height and
     the x_range and y_range. If x_sampling or y_sampling are supplied
-    the operation will ensure that a bin is no smaller than theminimum
+    the operation will ensure that a bin is no smaller than the minimum
     sampling distance by reducing the width and height when the zoomed
     in beyond the minimum sampling distance.
+
+    By default, the PlotSize operator is applied when this operation
```
Comment (philippjfr) [inline holoviews/operation/datashader.py]: >the PlotSize operator

"the PlotSize stream"

>be set to match the inner dimensions of the plot in which this is used.

We've standardized to the term "linked" would be good if you used that here.

Comment (jbednar) [inline holoviews/operation/datashader.py]: "inner dimensions of the linked plot"?

Comment (jlstevens) [inline holoviews/operation/datashader.py]: That seems reasonable to me.

Comment (philippjfr) [inline holoviews/operation/datashader.py]: Sounds good.

**Comment by philippjfr** [comment]
Looks good and since the tests passed and you just made docstring fixes I'll merge now.

**Pull request merged**